action.yml

```diff
@@ -1,29 +1,33 @@
-name: 'Retag Docker Image'
+name: 'Re-tag Docker Image'
 author: 'Koray Koska'
 branding:
   icon: 'message-square'
   color: 'gray-dark'
-description: 'Retag a Docker Image and push it to the registry'
+
+description: 'Re-tag a Docker Image and push it to the registry'
+
 inputs:
   registry:
     description: 'The registry to push the new tag later'
-    required: true
+    required: false
   name:
-    description: 'The name of the image you would like to retag'
-    required: true
+    description: 'The name of the image you would like to re-tag'
+    required: false
```

Comment:
Why is the name optional now? And the others too.

Reply:
I wanted to use `env` variables as alternatives to `with` and when they are required you cannot ignore them in the workflow. I guess `default` could be something to add here, but even then I am unsure if the users needs to still specify a `with` section or not.

Reply:
Added required back with defaults 😉  it works just fine